test/e2e/parallel/serviceaccount_test.go

```diff
@@ -57,47 +59,53 @@ func TestDoNotOverrideServiceAccount(t *testing.T) {
 		})
 		require.NoError(t, err)
 
-		// drop the SpaceRoles annotation from the namespace to trigger the reconciliation
-		_, err = member.UpdateNSTemplateSet(t, mur.Name, func(nsTmplSet *v1alpha1.NSTemplateSet) {
-			delete(nsTmplSet.Annotations, v1alpha1.LastAppliedSpaceRolesAnnotationKey)
+		// Update the namespace annotations & labels to trigger the reconciliation
+		_, err = member.UpdateNamespace(t, mur.Name, func(ns *corev1.Namespace) {
```

Comment:
```suggestion
		_, err = member.UpdateNamespace(t, nsName, func(ns *corev1.Namespace) {
```

Reply:
yeah, I know - already fixed that. Still doing & testing some changes locally  ;-)

Reply:
just wait a sec - got stuck in a meeting :smile:

Reply:
Sorry, I got email notifications that you pushed changes so I thought you were done :)